### DISCUSSION TITLE
Control theory question

### DISCUSSION THREAD

**Discussion by mhdadk** [ilayn/harold#100] [General] [answered]
Hi Ilhan,

This issue does not pertain to any specific problem, but I could not find a better way to contact you. Perhaps your email address i.polat@tue.nl (taken from one of your published IEEE papers) is more appropriate, but I'm not sure.

Anyway, onto my question. I read your comment [here](https://scicomp.stackexchange.com/questions/25730/computing-algebraic-riccati-inequality#comment43932_25730) on the scicomp.stackexchange.com site, and I found it to be very helpful in my research (I'm currently a PhD student). I have a question that is somewhat related to this comment that I posted [here](https://scicomp.stackexchange.com/q/43836/44044), and I would appreciate any help that you can offer. If you do not have the time, I completely understand.

Kind regards,
Mahmoud Abdelkhalek

**Comment by ilayn** [comment, marked as answer]
Hi @mhdadk Indeed this is not the place to have these convos since it is about a particular python package, and I am out of shape when it comes to control folks' theorem/proof game. Also I don't have access to the paper so not sure what is going on there and you need to have a lot of money to make me read it regardless. But just by guessing the context, the condition (1) is what is called the Kalman Filter LMI. It is everywhere online so you wouldn't need too much time on Google, here is one https://en.wikibooks.org/wiki/LMIs_in_Control/Applications/An_LMI_for_the_Kalman_Filter 

Then what you have is a solvability condition for a positive definite solution to a **nonstrict** Riccati inequality. Nonstrict AREs require that extra condition of controllability (stabilizability actually) to have a $P \succ 0$ otherwise you can only have $P \succeq 0$ at best. It's a bit lengthy to prove that but not very difficult so I'll leave it to you to find it in any LMI book but this is the main reason. 

Lastly, note that this part of control theory relies on essentially 3-4 tricks, schur complements, lyapunov argument, dissipative system LMIs, and very few more. So shuffle them in different order to get the "proof" of almost all "theorems". Good luck with your studies. 

PS: Please don't paste people's email on public space and respect their privacy unless they choose to do so. That email is not working so it is fine on this occasion.

**Reply by mhdadk** [reply]
Hi Ilhan, thank you for the response. I'm sorry I bothered you.
